src/Objects/DeviceFormatter.vala

```diff
@@ -105,6 +105,13 @@ namespace Formatter {
                         spawn_args += label;
                     }
                     break;
+                case Formatter.Filesystems.UDF:
+                    spawn_args = {"pkexec", "mkfs.udf", drive_identifier};
```

Comment:
Is this command available by default? Shouldn't we add any package to dependencies?

Reply:
Not sure what you mean. hfsprogs is not a dependency either.

Reply:
It is https://github.com/Djaler/Formatter/blob/deb-packaging-juno/debian/control#L14

Packaging resources are in another git branch